frontend/src/streamDetails/DetailedStreamStatistics/StatisticsOverviewCard.jsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { Grid } from "@material-ui/core";
+
+import DashboardCard from "../../general/dashboard/DashboardCard";
+import StreamStatisticsInfo from "../../model/StreamStatistics/StreamStatisticsInfo";
+import SimpleTable from "../../general/simpleTable/SimpleTable";
+
+export default function StatisticsOverviewCard(props) {
+  const { stats } = props;
+
+  const properties = {
+    "Stream ID": stats.id,
+    Time: stats.time,
+    Flow: stats.window.flow,
+    Congestion: stats.window.congestion,
+    Flight: stats.window.flight,
+    Latency: stats.link.rtt,
+    Bandwidth: stats.link.bandwidth,
+    "Maximum Bandwidth": stats.link.maxBandwidth
+  };
+
+  return (
+    <>
+      <DashboardCard title="Overview">
+        <Grid container>
+          <Grid item xs={12}>
+            <SimpleTable properties={properties} />
+          </Grid>
+        </Grid>
```

Comment:
@felixlapierre it `<Grid item/>` components need to be nested in `<Grid container/>` components to achieve proper behaviour. so this would normally be correct.
However, in #434, the Grid container was moved into the <DashboardCard/> component so you're not wrong. this outer grid can be removed

Reply:
348c00d 👍